src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -0,0 +1,195 @@
+<h3>Bestandsliste</h3>
+
+@code {
+
+// Parameter Items for asset list
+    [Parameter] public string[] Items { get; set; }
+
+    // TODO: remove the code for the initialization
+    private string[] _examples =
+    {
+        "Trockeneis", "Ethanol", "Natriumhydroxid", "Aceton", "Ammoniumchlorid", "Wasserstoffperoxid",
+        "Kaliumpermanganat", "Messzylinder", "Erlenmeyerkolben", "Reagenzglas"
+    };
+
+    protected override void OnInitialized()
+    {
+        Items = _examples;
+    }
+
+}
+
+
+@* grid with one column for items *@
+<div class="container text-center">
+    <div class="col">
+        @foreach (var item in Items)
+        {
+            <div
+                class="row custom-checkbox-container">
```

Comment:
🔧 Make this to single line element

Reply:
You can look at #22. Please use it everywhere in this file. In most places it is used, but not everywhere
In the Style guide are some settings that could help to format the file.